test/sequential/test-repl-persistent-history.js

```diff
@@ -74,35 +74,41 @@ const oldHistoryPath = path.join(fixtures, 'old-repl-history-file.json');
 const tests = [{
   env: { NODE_REPL_HISTORY: '' },
   test: [UP],
-  expected: [prompt, replDisabled, prompt]
+  expected: [prompt, replDisabled, prompt],
+  event: 'end'
```

Comment:
Are you sure this works? I don't think the repl actually emits this at all. Console.log something in `on('close',...)` / `on(opts.event,...)` and make sure it calls the same number as the number of tests.